cmd/atlas/internal/migrate/migrate.go

```diff
@@ -376,3 +381,12 @@ func DirURL(ctx context.Context, u *url.URL, create bool) (migrate.Dir, error) {
 	}
 	return d, err
 }
+
+// getEntDialect returns the corresponding ent dialect of the underlying Atlas client
+func (r *EntRevisions) getEntDialect() string {
+	if r.ac.Name == "libsql" {
+		return dialect.SQLite
+	}
+
+	return r.ac.Name
```

Comment:
Thanks for the contribution, but I prefer not to mix driver logic with cmdapi.

Drivers can implement their own RevisionsReadWriter, and I need to move this implementation out so the libsql driver one can reuse it.